client/src/frontend/components/composed/EntityMasthead/index.js

```diff
@@ -44,7 +44,7 @@ export default function Masthead({ entity }) {
           alt=""
         />
       )}
-      {logo && (
+      {!image && logo && (
```

Comment:
Larger open question here: how do we want to handle the masthead? Do we need to add a color selector and place for logo in the backend?